addon/adapters/file-provider.js

```diff
@@ -0,0 +1,7 @@
+import ApplicationAdapter from './application';
+
+export default ApplicationAdapter.extend({
```

Comment:
Is this adapter needed? The superclass impl. should already return 'files': https://github.com/emberjs/data/blob/v2.5.2/addon/adapters/json-api.js#L110